app/views/course/assessment/assessments/_form.html.slim

```diff
@@ -9,6 +9,7 @@
   = f.input :end_at
   = f.input :bonus_end_at
   = f.input :draft
+  = f.input :guided_mode, as: :select, collection: Course::Assessment.guided_modes.keys
```

Comment:
@lowjoel Think we should not allow changing this mode in `edit` ? what do you think ?